edn/src/edn.rustpeg

```diff
@@ -148,11 +148,11 @@ namespace_separator = "/"
 // TODO: Be more picky here
 // Keywords follow the rules of symbols, except they can (and must) begin with :
 // e.g. :fred or :my/fred. See https://github.com/edn-format/edn#keywords
-symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
-symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
+symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>.]
```

Comment:
This conflicts with the Clojure reader definition:

> Symbols begin with a non-numeric character and can contain alphanumeric characters and `*`, `+`, `!`, `-`, `_`, `'`, and `?` (other characters may be allowed eventually).
>
> `/` has special meaning, it can be used once in the middle of a symbol to separate the namespace from the name, e.g. `my-namespace/foo`. `/` by itself names the division function.
>
> `.` has special meaning - it can be used one or more times in the middle of a symbol to designate a fully-qualified class name, e.g. `java.util.BitSet`, or in namespace names. Symbols beginning or ending with `.` are reserved by Clojure. Symbols containing `/` or `.` are said to be 'qualified'.

That implies to me that either (a) we should document that our interpretation of EDN is diverging from Clojure's reader, or (b) we should special-case `.` and `...` as acceptable symbols.

I seem to remember that we did so…